freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020  Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from freshmaker import db
+from freshmaker.handlers import ContainerBuildHandler
+from freshmaker.events import BotasChangeStateEvent
+from freshmaker.models import Event
+from freshmaker.types import EventState
+
+
+class HandleBotasAdvisory(ContainerBuildHandler):
+    """
+    Handles event that was created by transition of an advisory filed by
+    BOTAS to SHIPPED_LIVE state
+    """
+    name = "HandleBotasAdvisory"
+    order = 0
+
+    def can_handle(self, event):
+        return isinstance(event, BotasChangeStateEvent)
+
+    def handle(self, event):
+        if event.dry_run:
+            self.force_dry_run()
+        self.event = event
+
+        # Generate the Database representation of `event`, it can be
+        # triggered by user, we want to track what happened
+        db_event = Event.get(db.session, event.msg_id)
```

Comment:
Will this event exist in the database here? This would be the first time that the event is seen by Freshmaker. 

You'll likely need to perform something like this:
https://github.com/redhat-exd-rebuilds/freshmaker/blob/40e36b671fe123f2e1427ef89b00fd4896f613f2/freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py#L70-L73

Reply:
So as I understood, it should be in DB, that's why there is only 'get()' without creating new event. But I am not sure. Maybe @gnaponie could help with this one?

Reply:
Well maybe I misunderstood something, but we were talking about situation, when 'FM rebuild operator/operand -> advisory created by BOTAS -> msg published to umb when that advisory moved to SHIPPED_LIVE state -> FM gets this message -> new event created -> botas handler handles that event'. But will that new event have information about builds made in before? So when I call something like  `new_db_event.builds`, will there be builds from previous event/advisory?

Reply:
@apaplaus If I understood correctly your point: the builds made for the operand/operator are not the same event/builds that will be done for the bundle images. Even though they are connected, they are still different container images.
The correct flow is: FM rebuild operator/operand -> advisory created by BOTAS -> msg published to umb when that advisory moved to SHIPPED_LIVE state -> FM gets this message (with this new botas handler that you wrote) -> new event created -> FM builds the bundle image

Reply:
Ok, so if they are connected and I can get nvrs in that way, how I am doing it now, everything is fine.